src/backend/ProcessLauncher.cpp

```diff
@@ -234,10 +234,10 @@ void ProcessLauncher::onProcessFinished(int exitcode, QProcess::ExitStatus exits
     afterRun();
 }
 
-void ProcessLauncher::beforeRun()
+void ProcessLauncher::beforeRun(const QString& gamePath)
 {
     TerminalKbd::enable();
-    ScriptRunner::run(ScriptEvent::PROCESS_STARTED);
+    ScriptRunner::run(ScriptEvent::PROCESS_STARTED, { args });
```

Comment:
Updating this line will fix the build error.

Reply:
Didn't realise I'd left that in here, my bad!